addons/editor-dark-mode/addon.json

```
@@ -1162,6 +1188,32 @@
       "name": "Scratch's default colors",
       "id": "scratch",
       "description": "The colors normally used by Scratch",
+      "values": {
+        "page": "#e5f0ff",
+        "primary": "#855cd6",
+        "highlightText": "#855cd6",
+        "menuBar": "#855cd6",
+        "popup": "#4d97ffe6",
+        "activeTab": "#ffffff",
+        "tab": "#d9e3f2",
+        "selector": "#e9f1fc",
+        "selector2": "#d9e3f2",
+        "selectorSelection": "#ffffff",
+        "accent": "#ffffff",
+        "input": "#ffffff",
+        "workspace": "#f9f9f9",
+        "categoryMenu": "#ffffff",
+        "palette": "#f9f9f9cc",
+        "fullscreen": "#ffffff",
+        "stageHeader": "#e8edf1",
+        "border": "#00000026",
+        "dots": true
+      }
+    },
+    {
+      "name": "Scratch's default colors (blue)",
```

They won't be default after the change.
```suggestion
      "name": "Scratch 3.0 (blue)",
```
Also I think it should be moved before Scratch 2 so the versions are in order, but it doesn't matter.